sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -128,6 +130,12 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
     }
     lifecycleState = LifecycleState.STATE_STARTED
 
+    mapboxMap.getStyle()?.let {
+      if (it != style) {
```

Comment:
is it correct way to compare style objects?

Reply:
Yes, we just compare that it's not the same reference since `equals` is not defined for Style